packages/spotlight/SpotlightContainerDecorator/SpotlightContainerDecorator.js

```diff
@@ -99,7 +101,23 @@ const defaultConfig = {
 	 * @memberof spotlight/SpotlightContainerDecorator.SpotlightContainerDecorator.defaultConfig
 	 * @public
 	 */
-	preserveId: false
+	preserveId: false,
+
+	/**
+	 * Restricts or prioritizes navigation when focus attempts to leave the container. It
+	 * can be either 'none', 'self-first', or 'self-only'. Specifying 'self-first' indicates that
+	 * elements within the container will have a higher likelihood to be chosen as the next
+	 * navigable element. Specifying 'self-only' indicates that elements in other containers
+	 * cannot be navigated to by using 5-way navigation - however, elements in other containers
+	 * can still receive focus by calling `Spotlight.focus(elem)` explicitly. Specifying 'none'
+	 * indicates there should be no restrictions when 5-way navigating the container.
```

Comment:
What do you think if
should be -> are

Reply:
I agree, I fixed.

Reply:
Changed my mind. should be looks better :)